foolbox/attacks/ead.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
+from __future__ import division
+
+import numpy as np
+import logging
+
+from .base import Attack
+from .base import call_decorator
+from ..utils import onehot_like
+
+
+class EADAttack(Attack):
+    """""Gradient based attack which uses an elastic-net regularization [1].
```

Comment:
looks like this are too many `""""""` ?

Reply:
Fixed in [a3720d6](https://github.com/bethgelab/foolbox/pull/336/commits/a3720d66efdcd09a16c242dabc2e2fd0b4d5c15e)